packages/firestore/src/local/simple_db.ts

```diff
@@ -147,15 +148,14 @@ export class SimpleDb {
       .catch(error => {
         // Abort the transaction if there was an
         // error.
-        transaction.abort();
-        return PersistencePromise.reject(error);
+        transaction.abort(error);
```

Comment:
Can you rewrap the comment while you're here?

Reply:
Done